bench/helpers.go

```diff
@@ -0,0 +1,161 @@
+// offers
+// https://github.com/topfreegames/offers
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2016 Top Free Games <backend@tfgco.com>
+
+package bench
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/satori/go.uuid"
+	"github.com/topfreegames/offers/models"
+	"gopkg.in/mgutz/dat.v2/dat"
+	runner "gopkg.in/mgutz/dat.v2/sqlx-runner"
+)
+
+func getRoute(url string) string {
+	return fmt.Sprintf("http://localhost:8888%s", url)
+}
+
+func createGames(db *runner.Connection, numberOfGames int) ([]*models.Game, error) {
```

Comment:
You could use the method `createGame` you wrote below inside this method, this way you can avoid the repeated code in lines 32-39